com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -101,8 +101,8 @@ public partial class UniversalRenderPipelineAsset : RenderPipelineAsset, ISerial
         ScriptableRenderer[] m_Renderers = new ScriptableRenderer[1];
 
         // Default values set when a new UniversalRenderPipeline asset is created
-        [SerializeField] int k_AssetVersion = 5;
-        [SerializeField] int k_AssetPreviousVersion = 5;
+        [SerializeField] int k_AssetVersion = 7;
+        [SerializeField] int k_AssetPreviousVersion = 7;
```

Comment:
Why 7 and not 6?

Reply:
The latest version now is 7 as I understand when we create the asset we want the latest one. Now why it was not 6 before, somebody forgot to bump it (So we were actually always upgrading new asset from 5 to 6.